packages/vm/src/types.ts

```diff
@@ -139,6 +139,11 @@ export interface VMOpts {
    * Use a custom EVM to run Messages on. If this is not present, use the default EVM.
    */
   evm?: EVMInterface
+
+  /**
+   * Activate VM debug logging
+   */
```

Comment:
The code docs on this should be adjusted to better describe what the option is doing.

When giving the whole option structure another thought: I think it is technically not true that we have this optional since we *do* have this actually activated by default (printing out CL debug statements). So we should rather model this with non-optional but defaulting to `true`.

I guess we also want to rename the option a bit to make it already clear by the option name what is being done here. Maybe we really go with something very explicit like `printCLDebugOutput`.

Reply:
(if someone has something shorter but similarly expressive let me know)